recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/display-manager.service

```diff
@@ -4,7 +4,7 @@ Requires=dbus.service weston.service
 After=dbus.service weston.service
```

Comment:
Also add Requires and After fields as following:

Requires=dbus.service weston-ready.service
After=dbus.service weston-ready.service